### ISSUE TITLE
Action button without toolbar at the top

### ISSUE THREAD

**Issue by stardothosting** [pdmfc/nova-action-button#25]
Because we have to register the action to use this button, it also puts the multi-select toolbar action option that is default for nova actions. Is there any way to disable that  but keep the button on per-row resources?

**Comment by skoyah** [comment]
Yeah, this is one of the caveats that It is referenced in the docs. I think a good solution might be using a dedicated route to handle the action instead of relying on the Nova route/controller. This one is a bit tricky because we need to mantain the same logic that Nova uses (gates, policies, permissions. validations, etc)

Would appreciate some help to develop this.

**Comment by skoyah** [comment]
This is also related with #3
